### PR TITLE
feat(Text): add Text component

### PR DESCRIPTION
**Pull request by chain-txz**

### CODE REVIEW

File: src/app/components/Text/__tests__/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import Text from '..';
+
+describe('<Tabs  />', () => {
```
Comment (angelia-yuqi-personal) [inline src/app/components/Text/__tests__/index.test.tsx:5]: name is wrong

File: src/app/components/Text/index.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  white-space: nowrap;
+  max-width: ${props => props.maxWidth};
+  display: ${props =>
+    props.maxWidth === undefined ? 'inherit' : 'inline-block'};
+  cursor: pointer;
+`;
+
+// note:
+// 1. maxWidth priority is higher than maxCount
+// 2. maxCount obly apply to string
```
Comment (angelia-yuqi-personal) [inline src/app/components/Text/index.tsx:30]: obly?

File: src/app/components/Text/index.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+// 3. if hoverValue if provide, use hoverValue as Tooltip text, other wise use children
```
Comment (angelia-yuqi-personal) [inline src/app/components/Text/index.tsx:31]: if hoverValue **if -> is** **provide -> provided**, otherwise xxxx

**Review by angelia-yuqi-personal — CHANGES_REQUESTED** [review]

File: src/app/components/Text/__tests__/index.test.tsx (hunk 1 above)
Comment (chain-txz) [reply src/app/components/Text/__tests__/index.test.tsx]: done

File: src/app/components/Text/index.tsx (hunk 2 above)
Comment (chain-txz) [reply src/app/components/Text/index.tsx]: done

File: src/app/components/Text/index.tsx (hunk 3 above)
Comment (chain-txz) [reply src/app/components/Text/index.tsx]: done

**Review by angelia-yuqi-personal — APPROVED** [review]

**Pull request merged**